components/Banner/examples/supplemental/MDCBannerView.h

```diff
@@ -18,15 +18,42 @@
 
 __attribute__((objc_subclassing_restricted)) @interface MDCBannerView : UIView
 
-@property(nonatomic, copy, nonnull) NSString *text;
+/**
+ A view that displays the text on a @c MDCBannerView.
+ @discussion Although this property is read-only, its properties are read/write. These properties can be used to configure the text shown on @c MDCBannerView.
+ For example:
+ @code
```

Comment:
Please remove these code snippets from the header documentation, here and throughout.

Reply:
Same reason as mentioned above.

Reply:
Talked offline. Addressed in c17c57d42ab6d485cc6570b2ccb6ded11981d656.